.codespell.exclude

```diff
@@ -95,3 +95,4 @@ planet "Sies Upi"
 		log "Minor People" "Torrey and Irene" `Torrey Dupont, an activist, is friends with SMDP militia NCO Irene Brower. You delivered a package for them.`
 		"Brin"
 		"exis"
+			`	"Oh yes. Acorn bread was a rare sight in Hai cuisine. It's too sweet, you see." He shifts his weight on the stool. "I'm old enough to remember hayipp bread, yur ki bread - you name it. Hayipp is a grain, similar to your human 'corn,' which creates a wholesome ruddy brown flour when milled. The bread was hearty, yet exquisite." You note the wistful look on his face. "Yur Ki is a milk, sourced from the livestock we raise - you may have seen some on Newhome; I'm told they resemble your human 'sheep.' Bread made from yur ki is rich and decadent. It is among my favorite foods to have for breakfast." He pauses for a moment and looks a bit downcast. "Since humans have joined us on our planets, all they want is acorn bread. The high demand has displaced all of the traditional breads that I grew up with. I miss them."`
```

Comment:
Oops.
```suggestion
			`	"Oh yes. Acorn bread was a rare sight in Hai cuisine. It's too sweet, you see." He shifts his weight on the stool. "I'm old enough to remember hayipp bread, yur ki bread - you name it. Hayipp is a grain, similar to your human 'corn,' which creates a wholesome ruddy brown flour when milled. The bread was hearty, yet exquisite." You note the wistful look on his face. "Yur ki is a milk, sourced from the livestock we raise - you may have seen some on Newhome; I'm told they resemble your human 'sheep.' Bread made from yur ki is rich and decadent. It is among my favorite foods to have for breakfast." He pauses for a moment and looks a bit downcast. "Since humans have joined us on our planets, all they want is acorn bread. The high demand has displaced all of the traditional breads that I grew up with. I miss them."`
```